app/assets/javascripts/discourse/app/templates/components/sidebar/section-link.hbs

```diff
@@ -1,63 +1,71 @@
 <div class="sidebar-section-link-wrapper">
-  <Sidebar::SectionLinkTo
-    @class={{concat @class (concat " sidebar-section-link sidebar-section-link-" @linkName)}}
-    @route={{@route}}
-    @query={{@query}}
-    @models={{if @model (array @model) (if @models @models (array))}}
-    @current-when={{@currentWhen}}
-    @title={{@title}}
-  >
-    {{#if @prefixValue }}
-      <span class="sidebar-section-link-prefix {{@prefixType}} {{@prefixCSSClass}}" style={{this.prefixCSS}}>
-        {{#if (eq @prefixType "image")}}
-          <img src={{@prefixValue}} class="prefix-image">
-        {{/if}}
+  {{#if @href}}
+    <a href={{@href}} rel="noopener noreferrer" target="_blank" class={{this.classNames}} title={{@title}}>
+      <span class="sidebar-section-link-content-text">
+        {{@content}}
+      </span>
+    </a>
+  {{else}}
+    <Sidebar::SectionLinkTo
+      @class={{this.classNames}}
+      @route={{@route}}
+      @query={{@query}}
+      @models={{if @model (array @model) (if @models @models (array))}}
```

Comment:
I know it's not part of this PR but I find this mega confusing to read :laughing: I think it would be better split out to a `this.resolvedModels` method or something, same as `classNames`, that could just do:

```javascript
resolvedModels() {
  return [this.model] if this.model;
  return this.models;
}
```

(that may not be quite right but you get the gist)

Reply:
Ah yes this is a good point, I've pushed a change to fix this. I think I did this previously when this was a template only component.